doc/src/tooltip/tooltip.html

```diff
@@ -47,8 +50,8 @@
               Dynamic Tooltip
             </button>
 
-            <p>I'm a paragraph with a <strong aubs-tooltip="text:Custom Class; class: tooltip, custom-tooltip">Tooltip</strong> with custom
-              class</p>
+            <p>I'm a paragraph with a <strong aubs-tooltip="text:Custom Class; class: tooltip, custom-tooltip">Tooltip</strong>              with custom class
```

Comment:
I like to keep lines shorter, I guess this was your formatter messing with the line breaks :p